tests/test_testing.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+import boost_histogram as bh
+import numpy as np
+
+import uhi.testing.indexing
+
+
+class TestAccess(uhi.testing.indexing.Indexing[bh.Histogram]):
+    @staticmethod
+    def make_histogram_1() -> bh.Histogram:
+        h1 = bh.Histogram(bh.axis.Regular(10, 0, 1))
+        h1[:] = [0, 2, 4, 6, 8, 10, 12, 14, 16, 18]
```

Comment:
I think this data should be defined in the uhi.testing.indexing.Indexing, right? So that if the testing algorithm changes the data can also be changed only on the UHI side.

Reply:
It could, but we also test setting values. I was thinking this could wait till we have serialization support, then we could provide a serialized histogram someone could use if they didn't want to set it up manually.